modules/ROOT/assets/attachments/sg.yaml

```diff
@@ -205,25 +205,34 @@ properties:
                     description: |
                       Enterprise Edition.
 
-                      High watermark for channel cache eviction (percent)
+                      High watermark for channel cache eviction (percent).
+
+                      When the cache size, determined by `max_number`, reaches the high watermark, the eviction process iterates through the cache to remove inactive channels.
                     default: 80
                   compact_low_watermark_pct:
                     type: integer
                     description: |
                       Enterprise Edition.
 
-                      Low watermark for channel cache eviction (percent)
+                      Low watermark for channel cache eviction (percent).
+
+                      When the cache size, determined by `max_number` returns to a value lower than `compact_low_watermark_pct`, the cache eviction process is stopped.
                     default: 60
                   max_number:
                     type: integer
                     description: |
                       Enterprise Edition.
 
-                      Maximum number of channel caches which will exist at any one point.
+                      Maximum number of channel caches which will exist at any one point. This property is used to determine the cache size (and the associated eviction watermarks `compact_low_watermark_pct`/`compact_high_watermark_pct`).
+
+                      The default value for this property is 50000. Along with the default channel `min_length` and `max_length` values, this would result in a memory usage under 1GB.
+
+                      The `max_number` value should be higher than the estimated number of replicating clients to optimize cache usage and minimize the number of cache misses.
```

Comment:
It's not actually the number of replicating clients, since multiple clients could be replicating the same channel(s).  

One option might be to mention the cache hit/miss stats here (cache.chan_cache_hits, cache.chan_cache_misses), and mention that this value can be tuned to optimize cache hits.